src/msw/mediactrl_am.cpp

```diff
@@ -1555,7 +1555,17 @@ void wxAMMediaEvtHandler::OnActiveX(wxActiveXEvent& event)
                 }
                 else
                 {
-                    m_amb->QueueStopEvent();
+                    if (false == m_bLoadEventSent)
+					{
+						m_amb->QueueStopEvent();
+					}
+
+					else
+					{
+		                m_bLoadEventSent	= false;
+						
+						m_amb->FinishLoad();
```

Comment:
This just doesn't make sense, you seem to have changed the meaning of `m_bLoadEventSent` flag without changing its name nor the comments describing it. A flag with this name must be set to `true`, not `false`, when `FinishLoad()` is called.

Reply:
Renaming m_bLoadEventSent to m_bLoadEventQueued would probably clear up the misunderstanding, but I wanted to fit the solution as closely as possible with the existing code base, changing as little as possible.

Reply:
Could you please update the PR to result in clear code and also add the comment(s) explaining what you wrote above?

Making as few changes as possible is certainly laudable, but it should be a secondary change, the primary ones should be to have correct and as simple/clear possible code.

TIA!